src/components/Pressable/PressableWithFeedback.js

```diff
@@ -39,25 +39,22 @@ const PressableWithFeedbackDefaultProps = {
 
 const PressableWithFeedback = forwardRef((props, ref) => {
     const propsWithoutWrapperStyles = _.omit(props, omittedProps);
-    const [disabled, setDisabled] = useState(props.disabled);
-    const [isPressed, setIsPressed] = useState(false);
+    const [isPressed, setPressed] = useState(false);
+    const [isPressedIn, setPressedIn] = useState(false);
```

Comment:
Can you revert the `isPressed`, `setIsPressed` logic? Instead find a new variable name for the other logic (you ask in Slack for some ideas)

Reply:
Do you mean to find a new name for both isPressed and isPressedIn? Or

isPressedIn -> isPressed
isPressed -> find a new name?

Reply:
(the second one)
We already have a isPressed state. Let's keep it
Find a new name for the other state

Reply:
Looks good